init.lua

```diff
@@ -231,7 +231,7 @@ minetest.register_chatcommand("s", {
 	func = function(name, param)
 		for _, toname in pairs(staff) do
 			minetest.chat_send_player(toname, minetest.colorize("#ff9900",
-				"«" .. name .. "» " .. param))
+				"#" .. name .. ": " .. param))
```

Comment:
Hmm, why have you used `#` and `:`?

Reply:
``<<player>>`` seemed too long and confusing to me, but i can change it to that if you wish, or to anything else if you have a better idea

Reply:
I think `<player> message` would suit this just fine, thanks to the message being colorized. Once CTF adds support for 5.1.0, we could then make use of the customisable chat message format (https://github.com/minetest/minetest/commit/d1c27c7e8037e9f438741ee5f7d3b6bda22d22a0).